### PR TITLE
Migrate ethlance to shadow-cljs

### PR DESCRIPTION
**Pull request by jpmonettas**

### CODE REVIEW

File: shadow-cljs.edn [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                   :infer-externs true
+                   :js-options {:js-package-dirs ["./ui/node_modules"]}
+                   :npm-deps false
+                   :output-dir "resources/public/js/"
+                   :optimizations :none
+                   :source-map true
+                   :source-map-timestamp true
+                   :closure-defines {"re_frame.trace.trace_enabled_QMARK_" true
+                                     goog.DEBUG true}}
+          ;; :dev-server {:target :node-script
```
Comment (fbielejec) [inline shadow-cljs.edn:108]: server builds are not managed by shadow?

**Review by fbielejec — COMMENTED** [review]
Looks like server build is not managed by shadow?

File: shadow-cljs.edn (hunk 1 above)
Comment (jpmonettas) [reply shadow-cljs.edn:108]: yeah it isn't, it will make this PR too big. It already contains district.ui.graphql removal + shadow-cljs for UI.

**Review by madvas — APPROVED** [review]

**Comment by fbielejec** [comment]
I get quite a few warnings and errors :+1: 

```
warning ethlance@0.0.1: "dependencies" has dependency "axios" with range "0.19.2" that collides with a dependency in "devDependencies" of the same name with version "0.19.0"
 
The required namespace "react" is not available, it was required by "reagent/core.cljs".
The namespace was provided via :foreign-libs which is not supported.
Please refer to https://shadow-cljs.github.io/docs/UsersGuide.html#cljsjs for more information.
You may just need to run:
  npm install react

Exception in thread "async-dispatch-7" java.lang.IllegalArgumentException: No implementation of method: :close! of protocol: #'clojure.core.async.impl.protocols/Channel found for class: nil
	at clojure.core$_cache_protocol_fn.invokeStatic(core_deftype.clj:583)
	at clojure.core$_cache_protocol_fn.invoke(core_deftype.clj:575)
	at clojure.core.async.impl.protocols$eval246$fn__247$G__235__252.invoke(protocols.clj:21)
	at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invokeStatic(ioc_macros.clj:984)
	at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invoke(ioc_macros.clj:980)
	at clojure.core.async$ioc_alts_BANG_$fn__5885.invoke(async.clj:421)
	at clojure.core.async$do_alts$fn__5816$fn__5819.invoke(async.clj:288)
	at clojure.core.async.impl.channels.ManyToManyChannel$fn__615$fn__616.invoke(channels.clj:95)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at clojure.core.async.impl.concurrent$counted_thread_factory$reify__484$fn__485.invoke(concurrent.clj:29)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
```

**Review by fbielejec — APPROVED** [review]

**Pull request merged**